pages/clubs.jsx

```diff
@@ -502,6 +502,64 @@ var ModalAddOrChangeYourClub = React.createClass({
   }
 });
 
+var ClubLists = React.createClass({
+  propTypes: {
+    clubs: React.PropTypes.array.isRequired,
+    username: React.PropTypes.string,
+    onDelete: React.PropTypes.func.isRequired,
+    onEdit: React.PropTypes.func.isRequired
+  },
+  componentWillMount: function() {
+    this.componentWillReceiveProps(this.props);
```

Comment:
(I was reading https://facebook.github.io/react/docs/component-specs.html)
Is it to make sure `this.state` will be updated for the initial `render` AND all the subsequent ones?